autoload/test/python.vim

```diff
@@ -1,4 +1,4 @@
 let test#python#patterns = {
   \ 'test':      ['\v^\s*%(async )?def (test_\w+)'],
-  \ 'namespace': ['\v^\s*class (\w+)'],
+  \ 'namespace': ['\v^\s*class (Test\w+)'],
```

Comment:
I think something like `'\v^\s*class .*Test'` could work, i.e. a class with `Test` in the name, or derived from something with `Test` in the name.
To be more strict `'\v^\s*class (Test\w+|.*TestCase.*)'` could be used maybe.

Reply:
I agree with the stricter version 👍